### ISSUE TITLE
something changed in the upstream openwrt github?

### ISSUE THREAD

**Issue by buzztiaan** [openwrt-zipit/openwrt-pxa-target#2]
i wanted to freshen up my zipit and setup a build environment again etc
but very early on i'm hitting errors

```
buzz@h81m:~/tmp$ mkdir blabla
buzz@h81m:~/tmp$ cd blabla/
buzz@h81m:~/tmp/blabla$  git clone https://github.com/openwrt/openwrt.git openwrt-zipit
Cloning into 'openwrt-zipit'...
remote: Enumerating objects: 22, done.
remote: Counting objects: 100% (22/22), done.
remote: Compressing objects: 100% (22/22), done.
remote: Total 454073 (delta 14), reused 0 (delta 0), pack-reused 454051
Receiving objects: 100% (454073/454073), 157.42 MiB | 11.08 MiB/s, done.
Resolving deltas: 100% (308167/308167), done.
buzz@h81m:~/tmp/blabla$  cd openwrt-zipit
buzz@h81m:~/tmp/blabla/openwrt-zipit$ git checkout 1e22c9b9eb691878156dfe32fb1e117737f1d248
fatal: reference is not a tree: 1e22c9b9eb691878156dfe32fb1e117737f1d248
buzz@h81m:~/tmp/blabla/openwrt-zipit$ 
```

**Comment by projectgus** [comment]
Looks like openwrt project rewrote some history when they merged with LEDE, but GitHub is still caching the old commit against that repo.

This repo contains the commit we were using: https://github.com/openwrt/archive

If you send a PR which updates either the repo to clone or the commit to use (with a new working commit from the "real" openwrt), I will happily merge it.

**Comment by mozzwald** [comment]
Fixed instructions with c18104ee5e2ba231228a5e86d25d96da369e7939
